modules/widgets/twitter-timeline.php

```diff
@@ -371,6 +374,7 @@ class="widefat"
 				name="<?php echo $this->get_field_name( 'height' ); ?>"
 				type="number" min="200"
 				value="<?php echo esc_attr( $instance['height'] ); ?>"
+				<?php echo $instance['tweet-limit'] ? 'disabled' : ''; ?>
```

Comment:
What do you think about relying on WordPress' `disabled` function here?
https://developer.wordpress.org/reference/functions/disabled/

Reply:
Good call!